Clockwise/InMemoryCircuitBreakerBroker.cs

```diff
@@ -51,10 +54,20 @@ public Task<CircuitBreakerStateDescriptor> GetLastStateAsync()
                         NotifyState();
                     }
                 }, expiry);
-                stateDescriptor = open;
+                SetCurrentState(open);
                 NotifyState();
             }
 
+            private void SetCurrentState(CircuitBreakerStateDescriptor newState)
+            {
+                if (stateDescriptor != newState)
+                {
+                    stateDescriptor = newState;
+                    logger.Event("CircuitBreakerTransition", ("circuitBreakerType", key),
```

Comment:
It seems misleading to log a state transition when the object is first constructed.

Reply:
not sure I follow, this is not used during first creation, only via signalFailure and signalSuccess